sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1345,6 +1345,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val LITERAL_PRECISE_PRECISION =
+    buildConf("spark.sql.literal.precisePrecision")
+      .internal()
+      .doc("When integral literals are used with decimals in binary operators, Spark will " +
+        "pick a precise precision for the literals to calculate the precision and scale " +
```

Comment:
`a precise precision` -> `the minimal precision required to represent the given value`?